### ISSUE TITLE
Bug: Pagecolor not effecting bottompart

### ISSUE THREAD

**Issue by shayant98** [Pyozer/introduction_screen#181]
**Describe the bug**
When setting a pagecolor on the PageViewModel it leaves the bottompart white. 
Setting a global color will not work for my case as all my pages have a different color

**To Reproduce**
1. Set a different page  color for more then 1 page
2. Show bottompart by enabling a button(skip, next, etc.)
3. bottom part is white

**Expected behavior**
Bottom part should also be the same color as the page

**Screenshots**

**Smartphone (please complete the following information):**
- Device: [e.g. Samsung 22 Ultra](https://gist.github.com/adamawolf/3048717)
- OS: [e.g. Android]
- Version [e.g.  Flutter 3.10]

**Additional context**

**Comment by Macacoazul01** [comment]
maybe related to #182.
Same request here to

**Comment by phankietit** [comment]
You can try using "globalBackgroundColor". Hope it helps you to solve the problem.

**Comment by ghenry** [comment]
Are we waiting for a new release to close this?

Thanks.

**Comment by Macacoazul01** [comment]
Nope, for @shayant98 to send a code and explain a little more the problem

**Comment by hirosz** [comment]
Hi! I have the same problem. I don't know what exactly makes that, but the workaround is to change in the library that part and subtract it from the bottom.
Line 624
 Positioned.fill(
                top: widget.bodyPadding.top,
                left: widget.bodyPadding.left,
                right: widget.bodyPadding.right,
                bottom: widget.bodyPadding.bottom
